application-gateway-logviewer-goaccess/ThirdPartyNotice

```diff
@@ -0,0 +1,309 @@
+THIRD PARTY SOFTWARE NOTICES AND INFORMATION
```

Comment:
remove this file - you can link to the official location in readme.md

Reply:
We have created this file keeping in mind the open source components we are using in this deployment, there is no separate official location for it as of yet. What is the issue in keeping the license file here?

Reply:
@bmoore-msft we were explicitly asked by CELA to keep the license details in this file - called ThirdPartyNotice. This was after a legal review of the open source components used in this template. See https://docs.opensource.microsoft.com/using/required-notice-template.html  I have also sent you a relevant email on this topic